backend/Repositories/SightingRepo.cs

```diff
@@ -12,5 +17,12 @@ public SightingRepo(WhaleSpottingDbContext context)
         {
             _context = context;
         }
+
+        public IEnumerable<Sighting> GetSightingsBySpeciesId(int speciesId)
+        {
+            return _context.Sightings
+                .Where(s => s.Species.Id == speciesId)
+                .OrderByDescending(s => s.SeenOn);
```

Comment:
Add some `Include()`s, and a `Where(s => s.ConfirmationStatus == ConfirmationStatus.Approved)`

Reply:
Update, thanks